credentials/tls/certprovider/provider.go

```diff
@@ -47,9 +47,9 @@ func Register(b Builder) {
 	m[b.Name()] = b
 }
 
-// getBuilder returns the Provider builder registered with the given name.
+// GetBuilder returns the Provider builder registered with the given name.
 // If no builder is registered with the provided name, nil will be returned.
-func getBuilder(name string) Builder {
+func GetBuilder(name string) Builder {
```

Comment:
This package is not internal, so this means the users can GetBuilder() directly.
Did we want users to use the store only?

Reply:
Switched back to not exporting, and using the internal package trick now.